conn.go

```diff
@@ -140,7 +137,6 @@ func getNeoBase(url string) (*neo4jBase, error) {
 
 	neoBase := neo4jBase{}
 	err = json.NewDecoder(res.Body).Decode(&neoBase)
-	io.Copy(ioutil.Discard, res.Body)
```

Comment:
So, whats happening in pseudocode with these statements:

1. make request  & bail if an error occurs
2. Read response into json decoder
3. Read again the empty response and discard it
4. Connection is closed at the end of the method anyway

There's not any reason to try to re-read the stream.